specification/elastic/resource-manager/Microsoft.Elastic/preview/2022-05-05-preview/elastic.json

```diff
@@ -0,0 +1,2302 @@
+      "post": {
+        "tags": [
+          "TrafficFilter"
+        ],
+        "summary": "Create and Associate IP traffic filter for the given deployment.",
+        "operationId": "createAndAssociateIPFilter_Create",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/MonitorNameParameter"
+          },
+          {
+            "in": "query",
+            "name": "ips",
+            "description": "List of ips",
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "name",
+            "description": "Name of the traffic filter",
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Success"
+          },
+          "default": {
+            "description": "Default error response.",
+            "schema": {
+              "$ref": "#/definitions/ResourceProviderDefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "createAndAssociateIPFilter_Create": {
+            "$ref": "./examples/IPTrafficFilter_Create.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Elastic/monitors/{monitorName}/createAndAssociatePLFilter": {
+      "post": {
+        "tags": [
+          "TrafficFilter"
+        ],
+        "summary": "Create and Associate private link traffic filter for the given deployment.",
+        "operationId": "createAndAssociatePLFilter_Create",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/MonitorNameParameter"
+          },
+          {
+            "in": "query",
+            "name": "name",
+            "description": "Name of the traffic filter",
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "privateEndpointGuid",
+            "description": "Guid of the private endpoint",
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "privateEndpointName",
+            "description": "Name of the private endpoint",
```

Comment:
move to body

Reply:
See previous comment [link](https://github.com/Azure/azure-rest-api-specs/pull/20791#discussion_r992778905)

Reply:
Same comment as above